jobmon_server/src/jobmon/server/web/models/task.py

```diff
@@ -59,7 +59,6 @@ def to_wire_as_swarm_task(self) -> tuple:
     resource_scales = Column(String(1000), default=None)
     fallback_queues = Column(String(1000), default=None)
     status = Column(String(1), ForeignKey("task_status.id"))
-    submitted_date = Column(DateTime)
```

Comment:
I'll note that we do use sqlalchemy to bootstrap the database schema for new databases, but the current prod database was built using the old SQL scripts in the jobmon repo. Just making sure we're aware that this change won't be reflected in the current schema

By the way, I thought we're only going to log Task Instance state transitions in the database, not task transitions. Curious if that's correct, if so we might still not know when a task was created - doesn't seem like useful info at the moment though